src/components/history/historyTypes.ts

```diff
@@ -0,0 +1,17 @@
+import { PostgrestError } from "@supabase/supabase-js";
+
+type userHistoryTypes = {
```

Comment:
Use PascalCase for type names

Reply:
@mansipadgelwar any reasons for suggesting using Pascal over Camel case for types?

Reply:
coding conventions